R/vimpact.R

```diff
@@ -433,3 +433,81 @@ get_impact_for_burden_estimate_set <- function(
     dplyr::select(country, year, age, burden_outcome, scenario, activity_type,
                   value)
 }
+
+#' Calculate impact from a recipe - requires a DB connection to montagu.
+#'
+#' This depends on the DB format of VIMC and so is for internal use only.
+#'
+#' @param con Connection to database
+#' @param recipe_path Path to file containing recipe for burden outcome
+#'    calculation. For more details see \code{vignette} vignette.
```

Comment:
Is there a better way to reference this vignette? It looks like can't have a straight link https://github.com/r-lib/roxygen2/issues/685#issuecomment-401132406

Reply:
if you write `vignette("name")` then pkgdown does a nice link and the user has something nice to work with